model-optimizer/extensions/ops/activation_test.py

```diff
@@ -67,3 +68,63 @@ def test_activation_elu_infer(self):
             self.assertEqual(res_shape[i], value)
         for i, value in enumerate(exp_value):
             self.assertAlmostEqual(res_value[i], value)
+
+    def test_activation_softplus_infer(self):
+        graph = build_graph(self.nodes_attributes,
+                            [
+                                ('node_1', 'activation_node'),
+                                ('activation_node', 'node_3')
+                            ],
+                            {
+                                'node_1': {
+                                    'value': np.array([-1.0, 0.0, 1.0, 20.0], dtype=np.float32)
+                                },
+                                'activation_node': {
+                                    'op': 'SoftPlus',
+                                    'operation': SoftPlus.operation,
+                                },
+                                'node_3': {
+                                    'value': None
+                                }
+                            })
+        graph.graph['layout'] = 'NCHW'
+        activation_node = Node(graph, 'activation_node')
+        SoftPlus.infer(activation_node)
+        exp_shape = np.array([227, 227, 227, 227])
```

Comment:
Why the expected shape is `[227, 227, 227, 227]`? It should be `[4]` according to the value provided. Please, fix other tests in this file.